src/main/kotlin/com/hypto/iam/server/configs/AppConfig.kt

```diff
@@ -45,7 +45,11 @@ data class AppConfig(val app: App, val server: Server, val database: Database, v
         val oldKeyTtl: Long,
         val secretKey: String,
         val signKeyFetchInterval: Long,
-        val cacheRefreshInterval: Long
+        val cacheRefreshInterval: Long,
+        val passcodeValidity: Long,
```

Comment:
Change variable names to include the unit. Ex - "passcodeValiditySeconds". Also the default is set to 600 (10 mins) which is too less. We should allow atleast 24 hrs.